be/src/exec/vectorized/streaming_aggregator.cpp

```diff
@@ -0,0 +1,401 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#include "exec/vectorized/streaming_aggregator.h"
```

Comment:
Add an annotation about the file. What's the difference between the aggregate_streaming_node.cpp and streaming_aggreator.cpp. It seems to be no connection. But the nonu is misleading.